README.md

```diff
@@ -1 +1,43 @@
-# leapp-report-analyzer
\ No newline at end of file
+# Leapp Report Analyzer
+
+Leapp Report Analyzer is a command-line tool for analyzing multiple Leapp reports and grouping systems by severity and entries.
+The tool processes JSON files containing Leapp reports and generates a JSON file and an HTML table displaying the results.
```

Comment:
I would link here the leapp report schema, or appropriate part of leapp documentation.

In first sentence then link leapp repo/docs.